src/restconfig/src/main/java/org/geoserver/rest/catalog/LayerGroupController.java

```diff
@@ -198,6 +203,11 @@ public boolean supports(MethodParameter methodParameter, Type targetType, Class<
 
     @Override
     public void configurePersister(XStreamPersister persister, XStreamMessageConverter converter) {
+        // add keyword object configuration
+        XStream xstream = persister.getXStream();
```

Comment:
Yes, but if you set it up in XStreamPersister instead of a REST controller, you'll get consistency between disk format and network format...